Makefile

```diff
@@ -113,10 +113,13 @@ $(BUILD_DIR)/mirror: $(BIO-ONTOLOGIES)
 # Component 2 --> Phenex data + TBox
 
 # Create Phenex data + TBox KB
-$(BUILD_DIR)/phenex-data+tbox.ofn: $(BUILD_DIR)/phenex-data-merged.ofn $(BUILD_DIR)/phenoscape-kb-tbox-classified.ofn
+$(BUILD_DIR)/phenex-data+tbox.ttl: $(BUILD_DIR)/phenex-data-merged.ofn $(BUILD_DIR)/phenoscape-kb-tbox-classified.ofn
 	$(ROBOT) merge \
     	-i $(BUILD_DIR)/phenex-data-merged.ofn \
     	-i $(BUILD_DIR)/phenoscape-kb-tbox-classified.ofn \
+    	-o phenex-data+tbox.ofn \
+    	&& $(ROBOT) convert \
+    	-i phenex-data+tbox.ofn \
     	-o $@
 
```

Comment:
Two things: with ROBOT you can chain commands, so you wouldn't need to save this file and the run again to `convert`. You can just add the `convert` step in the same execution. However the other thing is that you don't need the `convert` step. Just use the `.ttl` extension for the file to save from `merge`. ROBOT will guess the file format from that.